docs/projects/export-import/index.md

```diff
@@ -75,9 +75,14 @@ The Octopus Deploy data-model is a web, not a graph.  Some resources are shared
 
 Any environments which can be reached via the project will be included in the export.  This includes:
 
-- Environments included in any of the project's lifecycles 
-- Environments used to scope variables in any [library variable sets](/docs/projects/variables/library-variable-sets.md) connected to the project 
-- Environment restrictions defined on any accounts or certificates referenced by the project 
+- Environments included in any of the project's lifecycles.
+
+:::hint
```

Comment:
I don't know whether this is the right thing to use, but I've latched onto it since seeing it recently so I'm all in now lol

Reply:
I made a minor tweak to the wording and changed the hint to a warning and moved to below the bullet points.